components/Alert/Alert.tsx

```diff
@@ -4,10 +4,16 @@ import classNames from 'classnames';
 interface AlertProps {
   variant: 'notice' | 'error' | 'success';
   className?: string;
+  width?: string;
 }
 
-const Alert: FunctionComponent<AlertProps> = ({ className, children, variant }) => (
-  <div className={classNames(`alert alert--${variant}`, className)}>{children}</div>
+const Alert: FunctionComponent<AlertProps> = ({ className, children, variant, width = '50%' }) => (
+  <div className={classNames(`alert alert--${variant}`, className)}>
+    <style jsx>{`
+      width: ${width};
+    `}</style>
```

Comment:
Resist making changes to core widgets just to facilitate a single usage ... Alerts tend to take on the width of their containers, so this addition has little value in my opinion. If you want to limit the width for your alert, wrap it in a `div` and give it the width you desire.

Reply:
Removed unnecessary width prop from the alert component and also update localstorage with query strings for passing location data from the home page to location comparison page.